client/signup/config/step-components.js

```diff
@@ -28,7 +28,6 @@ const stepNameToModuleName = {
 	plans: 'plans',
 	'plans-new': 'plans',
 	'plans-business': 'plans',
-	'plans-ecommerce': 'plans',
```

Comment:
`Non-blocking / Nit`:

Could you explain more about which flow is being affected by the removal of `plans-ecommerce` here? I'm assuming one of the `start` flows?

Reply:
`/start` flow